addons/account/i18n/account.pot

```diff
@@ -3355,11 +3355,6 @@ msgstr ""
 msgid "Confirm"
 msgstr ""
 
-#. module: account
-#: model_terms:ir.ui.view,arch_db:account.view_bank_statement_search
-msgid "Confirmed"
-msgstr ""
-
```

Comment:
I don't think we're allowed removing existing terms in stable.

Reply:
Indeed, it might be useful for people who don't update the module